releasenotes/notes/deprecate-vqe-client-8d355440b6dcbdb4.yaml

```diff
@@ -0,0 +1,7 @@
+---
+deprecations:
+  - |
+    The :class:`.VQEClient` and its matching :class:`.VQERuntimeResult` are now
+    deprecated. Instead, users should migrate their code to use the Qiskit
+    Runtime Primitives. A guide on how to use this can be found
+    `here <https://qisk.it/algo_migration>`_.
```

Comment:
I think the idea is (as is with the other links of this style being used elsewhere), that these shortlinks will simply be updated to point to the correct resources once they are published. That means we will not need to change the code to do this. (@ElePT please correct me if I'm wrong here)

I will update the links used here to point more directly to VQE though :+1:

Reply:
I believe I said the same thing - my note was that until early May when Terra will be released, there will be window after this nature version is first released where it points to the .rst files.

> When the next Terra version is published this should be updated to point at the published html.

Bu "this" I meant the short url I had just mentioned should be updated, not the link here in the text in Nature in case that was the source of confusion.